plugins/status.js

```diff
@@ -0,0 +1,245 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const async = require('async');
+
+exports.register = function () {
+    this.outbound = require('./../outbound');
+    this.queue_dir = require('./../outbound/queue').queue_dir;
+}
+
+exports.hook_capabilities = function (next, connection) {
+
+    if (connection.remote.ip === '127.0.0.1' || connection.remote.ip === '::1') {
+        connection.capabilities.push('STATUS');
+    }
+
+    next();
+}
+
+exports.hook_unrecognized_command = function (next, connection, params) {
+    const self = this;
+
+    const result_process = function (err, result) {
```

Comment:
Now that we're in an es6 world with `const` and `let` not doing variable hoisting, we've deprecated variable expressions in favor of variable declarations (which are still hoisted).

```suggestion
    function result_process (err, result) {
```